.github/workflows/sonarqube.yml

```diff
@@ -8,30 +8,71 @@ on:
 
 permissions:
   pull-requests: read
+  actions: write
```

Comment:
it is necessary

Reply:
To upload the results on failure I thought so

Reply:
It should be enough if only read